Rotate LCD by 180 for stock Anycubic Predator

twagler opened the pull request:
Rotate LCD by 180 degrees for stock Anycubic Predator.

### Description

The stock Anycubic Predator LCD needs rotated by 180 degrees.  

### Benefits

This fix improves general usability of the LCD, or saves them from having to physically rotate it.

### Related Issues

None

twagler committed:
Update Configuration.h

Rotate LCD by 180 degrees for stock Anycubic Predator.

thisiskeithb commented on config/examples/delta/Anycubic/Predator/Configuration.h:
```diff
@@ -2396,6 +2396,7 @@
 // 480x320, 3.5", FSMC Stock Display from AnyCubic
 //
 #define ANYCUBIC_TFT35
+#define TFT_ROTATION TFT_ROTATE_180
```
There's already a `TFT_ROTATION` define further down in `Configuration.h`:

https://github.com/MarlinFirmware/Configurations/blob/8e78071ee2a69ab87734431feec4915f6978140c/config/examples/delta/Anycubic/Predator/Configuration.h#L2446

twagler committed:
Update Configuration.h

twagler commented:
@thisiskeithb You are correct, I missed that.  Updated to remove my initial insertion, and uncommented the one further down and changed to TFT_ROTATE_180.
